### DISCUSSION TITLE
Dad Jokes

### DISCUSSION THREAD

**Discussion by michaeljolley** [deepgram/community#61] [General help]
I thought my wife was joking when she said she'd leave me if I didn't stop singing "I'm A Believer"... Then I saw her face.

![Gif of the band The Monkeys making a disgusted face.](https://user-images.githubusercontent.com/1228996/207127029-6d471478-ce7b-4e89-b1d5-f7d66326fda1.gif)

**Comment by michaeljolley** [comment]
Do you know the best thing about booleans? Even if you're wrong, you're only off by a bit.

**Comment by jamonholmgren** [comment]
I was on a diabetes awareness site.

It asked me if I accept cookies.

Is that a trick question?
